src/main.ts

```diff
@@ -53,37 +34,98 @@ const command: Command & { __runner: any } = {
 			type: 'boolean',
 			default: false
 		});
-	},
-	run: async (helper: Helper, args: { pattern: string; dry: boolean }) => {
+	}
+
+	async getConfigs(fromVersion: number, toVersion: number = LATEST_VERSION): Promise<VersionConfig[]> {
+		const configs: VersionConfig[] = [];
+		for (let i = fromVersion + 1; i <= toVersion; ++i) {
+			configs.push((await import(`./v${i}/main`)).config as VersionConfig);
```

Comment:
Would there ever need to be a 4.1 type of upgrade (e.g. to replace a deprecated but not yet removed API)?

Reply:
At the moment, the plan is to provide upgrades for only major releases.